Audio_Steganography/TestingCode.py

```diff
@@ -0,0 +1,26 @@
+from Algos.Encrypt import Encrypt
```

Comment:
there is no folder named Algos. and therefore, this import statement does not work. You have created package for the Audio_Steganography folder.